### PR TITLE
membench:  adopt membench internally

### PR DESCRIPTION
**Pull request by deepfire**
1. intern a slightly modified copy of `cardano-node-membench`
2. drop the external membench

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -39,14 +32,32 @@
...
+    };
     ## This pin is to prevent workbench-produced geneses being regenerated each time the node is bumped.
     cardano-node-workbench = {
       url = "github:input-output-hk/cardano-node/44ac30fb04d02d41ba005ca5228db9b5e9b887d2";
       flake = false;
     };
+
+    cardano-mainnet-mirror.url = "github:input-output-hk/cardano-mainnet-mirror/nix";
+      # TODO, fix this
+      #db-analyser = network.haskellPackages.ouroboros-consensus-cardano.components.exes.db-analyser;
```
Comment (jbgi) [inline flake.nix:53]:
```suggestion
```

Comment (deepfire) [reply flake.nix]: @jbgi, done!

**Review by jbgi — APPROVED** [review]

**Review by jutaro — APPROVED** [review]

**Pull request merged**